mkdocs.yml

```
@@ -42,6 +42,10 @@ markdown_extensions:
         permalink: "¤"  # Adds a clickable permalink to each section heading
         toc_depth: 4
 
+extra_javascript:
+  - javascripts/mathjax.js
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)